src/pages/Feed/partials/Post/use-post-context-menu-items-explicit.ts

```diff
@@ -0,0 +1,58 @@
+import { IFeedItem } from '@feedfarm-shared/types';
```

Comment:
Note: I made two versions of this hook, one being more verbose and explicit (this one) and another, which I prefer, because it reads better.